### PR TITLE
Refactor IsExpired: Attach to Token struct

### PR DESCRIPTION
**Pull request by TheYkk**
#33 

With this refactor we can `IsExpired` function thread safe and token based

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -33,19 +33,6 @@ func ExampleNewBranca() {
...
-	if err != nil {
-		fmt.Printf("Error: %v\n", err)
-		return
-	}
-
-	// Set TTL for 1 minute
-	brc.SetTTL(60)
-}
-
 func ExampleBranca_IsExpired() {
```
Comment (andyone) [inline example_test.go:36]: You have removed this method but kept the usage example. Please rewrite it with an actual usage example of `token.IsExpired`.

**Review by andyone — CHANGES_REQUESTED** [review]

File: example_test.go (hunk 1 above)
Comment (TheYkk) [reply example_test.go]: Done, thank you for review.

**Comment by andyone** [comment]
Looks good, merged. Thanks!

**Review by andyone — APPROVED** [review]

**Comment by TheYkk** [comment]
Thank you very much, sorry for being agressif.

**Comment by andyone** [comment]
@TheYkk np

**Pull request merged**